sdk/src/main/java/io/dapr/client/DaprClientGrpc.java

```diff
@@ -332,6 +332,9 @@
     byte[] data = payload == null ? null : payload.toByteArray();
     T value = stateSerializer.deserialize(data, type);
     String etag = response.getEtag();
+    if (etag.equals("")) {
```

Comment:
Bulk state needs this change too. See method above this one.

Reply:
good catch. missed that ..

Reply:
Done